src/Pages/heohyesung/Login/Login.js

```diff
@@ -1,14 +1,85 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
 
 class LoginHeo extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      pw: "",
+      backgroundcolor: "rgb(192, 223, 253)",
+    };
+  }
+
+  handleIdInput = (e) => {
+    this.setState({
+      id: e.target.value,
+    });
+  };
+
+  handlePwInput = (e) => {
+    this.setState({
+      pw: e.target.value,
+    });
+  };
+
+  loginKeyUp = () => {
+    if (this.state.id.includes("@") && this.state.pw.length >= 5) {
+      this.setState({
+        backgroundcolor: "rgb(8, 150, 247)",
+      });
+    } else {
+      this.setState({
+        backgroundcolor: "rgb(192, 223, 253)",
+      });
+    }
```

Comment:
- 위에 드린 리뷰 내용에 상관없이 해당 코드만 보고 리팩토링 리뷰 드리겠습니다.
- 반복되어 사용되는 코드를 줄이며 삼항연산자를 활용할 수 있습니다. 
- 구조분해 할당도 해주시면 더욱 깔끔한 코드가 되겠네요.
```suggestion
    const { id, pw } = this.state;
    this.setState({
      backgroundcolor:
        id.includes("@") && pw.length >= 5
          ? "rgb(8, 150, 247)"
          : "rgb(192, 223, 253)",
    });
```

Reply:
네 수정하도록하겠씁니다